### ISSUE TITLE
Incomplete Flip in 3.1.5 R2

### ISSUE THREAD

**Issue by rudafm** [MegaPirateNG/ardupilot-mpng#167]
Everything works OK except the Flip.It is not fully 360 degrees but approximately 350 degrees, so after Flip the Quad goes to the left  and I have to correct the roll...
/ Stabilize Mode /.
Please, have You any idea what is wrong ?

**Comment by ar99** [comment]
Yes, the same issue!. my tricopter go to the left I didn´t know why!
MPNG 3.1.5

**Comment by Floyer007** [comment]
I've the same issue.

My quadrocopter turns left/right (depends on the random flip-direction) after the flip.
I have to correct it with roll, but after releasing the roll-stick it turns in the same direction after the flip again.

Possible solution for this bug:
Land and re-arm the copter and everything works fine again.
But while flying after the flip it's very difficult to steer the copter in the direction I want.

It reacts like it's wrong trimmed after the flip or like rudafm said it has a wrong offset.
